src/main.ts

```diff
@@ -3,5 +3,10 @@ export * from './compositions'
 export * from './errors'
 export * from './services'
 export * from './types'
+
+export type { ParamStart, ParamEnd, ExtractParamName, ExtractPathParamType, ExtractRouteParamTypes, ExtractParamTypes, ExtractParamType, MergeParams } from './types/params'
+export { paramStart, paramEnd, isParamGetter, isParamGetSet } from './types/params'
+export type { CreateRouteOptions } from './types/createRouteOptions'
```

Comment:
A lot of these I wouldn't expect to be exported. Probably just the `CreateRouteOptions` type? Curious what your thoughts are here.

Reply:
I moved the top 2 lines from barrel of @/types without changing what's exported.

That being said I don't disagree. I'll remove lines 7 and 8